src/sasctl/_services/folders.py

```diff
@@ -154,21 +154,21 @@ def create_path(cls, folder, description=None):
             Details of newly-created folder
 
         """
-        if isinstance(folder, str) and folder.count("/") >= 1:
-            # Path must include a leading "/"
-            if not folder.startswith("/"):
-                folder = f"/{folder}"
+        folder = str(folder)
+        # Path must include a leading "/"
+        if not folder.startswith("/"):
+            folder = f"/{folder}"
         else:
             raise ValueError(
                 "Invalid folder parameter. The complete path should be provided!")
 
         path = folder.split("/")
         level = 1
-        while level < len(path):
+        for level in range(2, len(path)+1):
             level += 1
```

Comment:
remove `level = 1` and `level += 1` since range() will now take care of incrementing values.